From 5add5a18d2702d1749283e76d9581937db15b0f7 Mon Sep 17 00:00:00 2001
From: Jukka Laitinen <jukka.laitinen@intel.com>
Date: Thu, 11 May 2017 11:52:37 +0300
Subject: [PATCH 01/13] VENDOR: ASoC: intel: skylake: Add DSP FW load for Bxt-M

This adds the missing fw initialization to the DSP ops lookup table.

This was missing in the commit c375476550008d9a77f1bfbb36ec3ed4afcdd745
for newer kernels, where the DPS object creation and fw load has been
split.

Signed-off-by: Jukka Laitinen <jukka.laitinen@intel.com>
---
 sound/soc/intel/skylake/skl-messages.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/skylake/skl-messages.c b/sound/soc/intel/skylake/skl-messages.c
index b5d9d99..488e6ab 100644
--- a/sound/soc/intel/skylake/skl-messages.c
+++ b/sound/soc/intel/skylake/skl-messages.c
@@ -224,6 +224,7 @@ static const struct skl_dsp_ops dsp_ops[] = {
 		.id = 0x1a98,
 		.loader_ops = bxt_get_loader_ops,
 		.init = bxt_sst_dsp_init,
+		.init_fw = bxt_sst_init_fw,
 		.cleanup = bxt_sst_dsp_cleanup
 	},
 };
-- 
2.7.4

